### ISSUE TITLE
[Error] [brightcove:new] Unable to download webpage: timed out (caused by TransportError('timed out'))

### ISSUE THREAD

**Issue by momogood168** [yt-dlp/yt-dlp#10028]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Japan

### Provide a description that is worded well enough to be understood

upon download from below link https://tver.jp/episodes/epy0d467e7
[brightcove:new] Unable to download webpage: timed out (caused by TransportError('timed out'))

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
D:\yt-dlp>yt-dlp https://tver.jp/episodes/epy0d467e7 -vU
[debug] Command-line config: ['https://tver.jp/episodes/epy0d467e7', '-vU']
[debug] Encodings: locale cp932, fs utf-8, pref cp932, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.10.13 [b634ba742] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2022-04-07-git-607ecc27ed-full_build-www.gyan.dev (setts), ffprobe 2022-04-07-git-607ecc27ed-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.19.0, brotli-1.1.0, certifi-2023.07.22, mutagen-1.47.0, sqlite3-3.35.5, websockets-11.0.3
[debug] Proxy map: {}
[debug] Extractor Plugins: SamplePluginIE
[debug] Post-Processor Plugins: SamplePluginPP
[debug] Loaded 1891 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2024.04.09, Current version: stable@2023.10.13
Current Build Hash: 13d6a04ef6678dc61cb6e2d42eb53e69486bd2f52b8e5d778db029ccd4c600b4
[debug] Downloading _update_spec from https://github.com/yt-dlp/yt-dlp/releases/latest/download/_update_spec
Updating to stable@2024.04.09 ...
[debug] Downloading yt-dlp.exe from https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp.exe
[debug] Downloading SHA2-256SUMS from https://github.com/yt-dlp/yt-dlp/releases/latest/download/SHA2-256SUMS
Updated yt-dlp to stable@2024.04.09
[debug] Restarting: yt-dlp "https://tver.jp/episodes/epy0d467e7" -vU
[debug] Command-line config: ['https://tver.jp/episodes/epy0d467e7', '-vU']
[debug] Encodings: locale cp932, fs utf-8, pref cp932, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.04.09 from yt-dlp/yt-dlp [ff0779267] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2022-04-07-git-607ecc27ed-full_build-www.gyan.dev (setts), ffprobe 2022-04-07-git-607ecc27ed-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.31.0, sqlite3-3.35.5, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Extractor Plugins: SamplePluginIE
[debug] Post-Processor Plugins: SamplePluginPP
[debug] Loaded 1811 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2024.04.09 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2024.04.09 from yt-dlp/yt-dlp)
[TVer] Creating session
[TVer] Extracting URL: https://tver.jp/episodes/epy0d467e7
[TVer] epy0d467e7: Downloading JSON metadata
[TVer] epy0d467e7: Downloading JSON metadata
[brightcove:new] Extracting URL: http://players.brightcove.net/4394098882001/default_default/index.html?videoId=ref:fea96d8522684620905bb939a6f31a77#__youtubedl_smuggle=%7B%22geo_countries%22%3A+%5B%22JP%22%5D%7D
[debug] Using fake IP 133.77.175.154 (JP) as X-Forwarded-For
[brightcove:new] ref:fea96d8522684620905bb939a6f31a77: Downloading JSON metadata
WARNING: [brightcove:new] Unable to download JSON metadata: HTTPConnectionPool(host='players.brightcove.net', port=80): Read timed out. (read timeout=20.0)
[brightcove:new] ref:fea96d8522684620905bb939a6f31a77: Downloading webpage
ERROR: [brightcove:new] Unable to download webpage: HTTPConnectionPool(host='players.brightcove.net', port=80): Read timed out. (read timeout=20.0) (caused by TransportError("HTTPConnectionPool(host='players.brightcove.net', port=80): Read timed out. (read timeout=20.0)"))
  File "yt_dlp\extractor\common.py", line 734, in extract
    ie_result = self._real_extract(url)
  File "yt_dlp\extractor\brightcove.py", line 911, in _real_extract
    policy_key = extract_policy_key()
  File "yt_dlp\extractor\brightcove.py", line 880, in extract_policy_key
    webpage = self._download_webpage(
  File "yt_dlp\extractor\common.py", line 1176, in _download_webpage
    self._downloader.deprecation_warning('timeout argument is deprecated in InfoExtractor._download_webpage')
  File "yt_dlp\extractor\common.py", line 1127, in download_content
    'impersonate': impersonate,
  File "yt_dlp\extractor\adobepass.py", line 1366, in _download_webpage_handle
    return super(AdobePassIE, self)._download_webpage_handle(
  File "yt_dlp\extractor\common.py", line 954, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data,
  File "yt_dlp\extractor\common.py", line 903, in _request_webpage
    raise ExtractorError(errmsg, cause=err)

  File "urllib3\connectionpool.py", line 537, in _make_request
  File "urllib3\connection.py", line 466, in getresponse
  File "http\client.py", line 1344, in getresponse
  File "http\client.py", line 307, in begin
  File "http\client.py", line 268, in _read_status
  File "socket.py", line 669, in readinto
socket.timeout: timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "requests\adapters.py", line 486, in send
  File "urllib3\connectionpool.py", line 847, in urlopen
  File "urllib3\util\retry.py", line 445, in increment
  File "urllib3\util\util.py", line 39, in reraise
  File "urllib3\connectionpool.py", line 793, in urlopen
  File "urllib3\connectionpool.py", line 539, in _make_request
  File "urllib3\connectionpool.py", line 370, in _raise_timeout
urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='players.brightcove.net', port=80): Read timed out. (read timeout=20.0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "yt_dlp\networking\_requests.py", line 313, in _send
    session.mount('https://', http_adapter)
  File "requests\sessions.py", line 589, in request
  File "requests\sessions.py", line 703, in send
  File "requests\adapters.py", line 532, in send
requests.exceptions.ReadTimeout: HTTPConnectionPool(host='players.brightcove.net', port=80): Read timed out. (read timeout=20.0)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "yt_dlp\extractor\common.py", line 890, in _request_webpage
    return self._downloader.urlopen(self._create_request(url_or_request, data, headers, query, extensions))
  File "yt_dlp\YoutubeDL.py", line 4131, in urlopen
  File "yt_dlp\networking\common.py", line 115, in send
    self._print_verbose(f'Sending request via "{handler.RH_NAME}"')
  File "yt_dlp\networking\_helper.py", line 208, in wrapper
    return func(self, *args, **kwargs)
  File "yt_dlp\networking\common.py", line 335, in send
    if not isinstance(request, Request):
  File "yt_dlp\networking\_requests.py", line 337, in _send
    stream=True
yt_dlp.networking.exceptions.TransportError: HTTPConnectionPool(host='players.brightcove.net', port=80): Read timed out. (read timeout=20.0)
```

**Comment by bashonly** [comment]
I can't reproduce the issue. Do you use pihole or any sort of firewall? Does it play in browser?

**Comment by momogood168** [comment]
No firewall installed and the video can play in browser. But I need to use vpn to open it

**Comment by bashonly** [comment]
Closing in favor if #10199

Will be fixed by #10202

**Comment by zejjnt** [comment]
Sorry to necro but this issue ~persists~ reapperaed recently, I'm running the latest nightly. It was not fixed in #10202.
Video plays fine in browser. No blockers of any kind in front of the machine.

**Comment by bashonly** [comment]
@zejjnt open a new issue with complete verbose output
